core/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -507,8 +507,13 @@ export async function assertIdentityTransformation(
   expect(modelsOnlyInSourceAsInvariant).to.have.length(0);
   expect(onlyInTargetModels).to.have.length(0);
 
-  const makeRelationKey = (rel: any) => `${rel.SourceECInstanceId}\x00${rel.TargetECInstanceId}`;
-  const query: Parameters<IModelDb["query"]> = ["SELECT * FROM bis.ElementRefersToElements", undefined, { rowFormat: QueryRowFormat.UseECSqlPropertyNames }];
+  const makeRelationKey = (rel: any) =>
```

Comment:
@MichaelBelousov IModelTransformerUtils.ts contained 10 conflicts. Please verify I resolved them correctly.

Reply:
if the typescript builds (I didn't manually check that the imports are all correct), then `IModelTransformerUtil.ts` is correct